.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,48 @@
+name: Compute Benchmarks
```

Comment:
either remove `performance.yml` now or update `performance.yml` content to use the reusable workflow and remove this new `benchmarks.yml` - we just need one of these 2 files